apps/condo/domains/common/hooks/useSearchByPhoneModal.tsx

```diff
@@ -8,39 +8,34 @@ import React, { useCallback, useMemo, useState } from 'react'
 import { PhoneInputProps } from 'react-phone-input-2'
 
 import { useIntl } from '@open-condo/next/intl'
+import { Modal } from '@open-condo/ui'
 
 import Select from '@condo/domains/common/components/antd/Select'
 import { Button } from '@condo/domains/common/components/Button'
 import { GraphQlSearchInput, SearchComponentType } from '@condo/domains/common/components/GraphQlSearchInput'
-import { Modal } from '@condo/domains/common/components/Modal'
+import { useLayoutContext } from '@condo/domains/common/components/LayoutContext'
 import { PhoneInput } from '@condo/domains/common/components/PhoneInput'
-import { colors } from '@condo/domains/common/constants/style'
 import {
     ClientType,
     mapSearchItemToOption,
     redirectToForm,
 } from '@condo/domains/contact/utils/clientCard'
 
 
-const StyledModal = styled(Modal)`
+const StyledModal = styled(Modal)<{ isSmall }>`
```

Comment:
You can override centered behaviour here for you custom usage (and even better to do it in css/less module.